src/models/transfers.js

```diff
@@ -461,10 +453,26 @@ function * getFulfillment (transferId) {
   return fulfillment.getDataExternal()
 }
 
+function * resendNotification (transferId) {
```

Comment:
Do you see any benefit to emitting an event to indicate there was an error (in the case that the transfer is not found, for example)? My understanding is that this method will be used when a client (such as RC) requests the state of a particular transfer. If the client has a transfer that the ledger does not, that indicates a pretty serious data integrity issue.